### PR TITLE
feat(backend): add backend extension template

### PR DESCRIPTION
**Pull request by gregorycat**
MP-1250

### CODE REVIEW

File: template-backend-extension/src/widget/WidgetGlobalSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { Link } from '@lumx/react';
+import { NotificationsProvider, PredefinedErrorBoundary, useLanguage } from 'lumapps-sdk-js';
+
+import messagesEn from '../translations/en.json';
+import messagesFr from '../translations/fr.json';
+
+type WidgetGlobalSettings = import('lumapps-sdk-js').GlobalSettingsComponent<any>;
+
+/**
+ * Render the widget Picsum settings form.
```
Comment (LaurentMox) [inline template-backend-extension/src/widget/WidgetGlobalSettings.tsx:13]: Not the picsum

File: template-backend-extension/src/widget/WidgetGlobalSettings.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * @param {Object} props The settings component properties.
+ */
+const IntlWidgetGlobalSettings: WidgetGlobalSettings = ({ properties = {}, exportProp }) => {
+    const contactMail = '';
+    const subjectMail = '';
```
Comment (LaurentMox) [inline template-backend-extension/src/widget/WidgetGlobalSettings.tsx:19]: Unused

File: template-backend-extension/src/widget/WidgetGlobalSettings.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ *
+ * @param {Object} props The settings component properties.
+ */
+const IntlWidgetGlobalSettings: WidgetGlobalSettings = ({ properties = {}, exportProp }) => {
+    const contactMail = 'contact@mail.com';
+    const subjectMail = 'Extension activation';
+
+    return (
+        <div>
+            <p className="lumx-typography-body2">
```
Comment (germainruget) [inline template-backend-extension/src/widget/WidgetGlobalSettings.tsx:23]: LGTM.
Minor issue, it seems there is some prettier issues in this file. Except that, seems good to me

**Review by germainruget — DISMISSED** [review]

File: template-backend-extension/src/widget/WidgetGlobalSettings.tsx (hunk 3 above)
Comment (germainruget) [reply template-backend-extension/src/widget/WidgetGlobalSettings.tsx:23]: Tests OK : 
Could "install" the backend template : OK
Everything load correctly : OK
Only access to Global settings : OK

**Review by germainruget — APPROVED** [review]

**Pull request merged**